crates/lib/src/rules/convention/CV03.rs

```diff
@@ -0,0 +1,182 @@
+use crate::core::config::Value;
+use crate::core::parser::segments::base::{ErasedSegment, SymbolSegment};
+use crate::core::rules::base::{Erased, ErasedRule, LintFix, LintResult, Rule};
+use crate::core::rules::context::RuleContext;
+use crate::core::rules::crawlers::{Crawler, SegmentSeekerCrawler};
+use crate::utils::functional::context::FunctionalContext;
+
+#[derive(Debug, Clone)]
+pub struct RuleCV03 {
+    select_clause_trailing_comma: String,
+}
+
+impl Default for RuleCV03 {
+    fn default() -> Self {
+        RuleCV03 { select_clause_trailing_comma: "reuired".to_string() }
```

Comment:
I am going to guess this is a typo 

```suggestion
        RuleCV03 { select_clause_trailing_comma: "required".to_string() }
```